gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GenerateResourceAccessorsTask.kt

```diff
@@ -31,9 +43,6 @@ internal abstract class GenerateResourceAccessorsTask : IdeaImportTask() {
     @get:Input
     abstract val makeAccessorsPublic: Property<Boolean>
 
-    @get:Input
-    abstract val generateResourceContentHashAnnotation: Property<Boolean>
-
```

Comment:
why did you delete it?

Reply:
It is inherited from the interface. What is wrong with such properties?